File: 2.0/java/contacts/addContactListItems.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.Contact;
import com.callfire.api.client.api.contacts.model.request.AddContactListItemsRequest;

import java.util.Arrays;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");

        Contact contact1 = new Contact();
        contact1.setFirstName("Alice");
        contact1.setLastName("Moore");
        contact1.setHomePhone("12135551126");

        Contact contact2 = new Contact();
        contact1.setFirstName("Bob");
        contact1.setLastName("Smith");
        contact1.setHomePhone("12135551127");
        contact1.getProperties().put("age", "30");

        AddContactListItemsRequest request1 = AddContactListItemsRequest.<Contact>create()
            .contactListId(45006708003L)
            .contacts(Arrays.asList(contact1, contact2))
            .build();
        client.contactListsApi().addListItems(request1);

        AddContactListItemsRequest request2 = AddContactListItemsRequest.<String>create()
            .contactListId(45006708003L)
            .contactNumbersField("workPhone")
            .contacts(Arrays.asList("12132212384", "12136612355", "12133312300"))
            .build();
        client.contactListsApi().addListItems(request2);

        AddContactListItemsRequest request3 = AddContactListItemsRequest.<Long>create()
            .contactListId(45006708003L)
            .contacts(Arrays.asList(800634185003L, 800734186003L, 800834187003L, 800984185003L))
            .build();
        client.contactListsApi().addListItems(request3);
    }
}

File: 2.0/csharp/contacts/addContactListItems.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;
using CallfireApiClient.Api.Contacts.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request1 = new AddContactListContactsRequest<Contact>
        {
            Contacts = new List<Contact>
            {
                new Contact
                {
                    FirstName = "Alice",
                    LastName = "Moore",
                    HomePhone = "12135551126"
                },
                new Contact
                {
                    FirstName = "Bob",
                    LastName = "Smith",
                    HomePhone = "12135551127",
                    Properties = new Dictionary<string, string>
                    {
                        {"age", "30"}
                    }
                }
            },
            ContactListId = 45006708003
        };
        client.ContactListsApi.AddListItems(request1);

        var request2 = new AddContactListContactsRequest<string>
        {
            Contacts = new List<string> {"12132212384", "12136612355", "12133312300"},
            ContactNumbersField = "workPhone",
            ContactListId = 45006708003
        };
        client.ContactListsApi.AddListItems(request2);

        var request3 = new AddContactListContactsRequest<long>
        {
            Contacts = new List<long> {800634185003, 800734186003, 800834187003, 800984185003},
            ContactListId = 45006708003
        };
        client.ContactListsApi.AddListItems(request3);
    }
}
